getting-started-ng/1.md

```diff
@@ -5,7 +5,7 @@
 Welcome to the NativeScript getting started guide. In this first lesson, you’ll learn what NativeScript is, how to use this Playground environment, and how to get your first app on your device.
 
 > **TIP**:
-> * If you run into problems during this guide, the [NativeScript community forum](https://discourse.nativescript.org/) is a great place to reach out for help.
+> * This tutorial is for using NativeScript with Angular. If you'd prefer to stick with plain JavaScript, check out [our tutorial on NativeScript Core](https://play.nativescript.org/?template=groceries-js&tutorial=groceries-js), the dependency-free version of NativeScript.
```

Comment:
When referring to Playground content, it would be better to use relative links (`/?template=groceries-js&tutorial=groceries-js` instead of `https://play.nativescript.org/?template=groceries-js&tutorial=groceries-js`) in order to account for development and staging environments.